tool/speed.cc

```diff
@@ -163,6 +163,13 @@ static uint64_t g_timeout_seconds = 1;
 static std::vector<size_t> g_chunk_lengths = {16, 256, 1350, 8192, 16384};
 
 static bool TimeFunction(TimeResults *results, std::function<bool()> func) {
+#if defined(BORINGSSL_FIPS)
```

Comment:
Why does this have to be for fips build only?

Reply:
Good point, anything could have this cold start behavior. I'll update it.